types/mongodb/test/collection/updateX.ts

```diff
@@ -0,0 +1,28 @@
+import { connect, MongoError, Cursor } from 'mongodb';
+import { connectionString } from '../index';
+
+// collection.updateX tests
+async function run() {
+  const client = await connect(connectionString);
+  const db = client.db('test');
+  const collection = db.collection('test.find');
+
+  // test with collection type
+  interface TestModel {
+    stringField: string;
+    numberField?: number;
+    fruitTags: string[];
+  }
+  const collectionT = db.collection<TestModel>('testCollection');
+  collectionT.updateOne(
+    { stringField: 'hola' },
+    {
+      $addToSet: {
+        fruitTags: 'Orange',
+      },
+      $pull: {
+        fruitTags: 'Lemon',
+      },
+    },
+  );
+}
```

Comment:
can you also add `updateMany`?

Reply:
Sure I just reorganized the current tests (mostly without any modifications) in the first commit. Now I'm going to add new tests 🙂

Reply:
Oh ok. Let us know when you are done, so I'll recheck it